### PR TITLE
mostly client mappings

### PR DESCRIPTION
**Pull request by eristhea**
new mappings? in this economy?

discussion question: should we consider breaking up the "every block is in `net.minecraft.block`" convention for absolutely wild stuff like putting sign blocks in `net.minecraft.block.sign`?

### CODE REVIEW

File: mappings/net/minecraft/client/multiplayer/prediction/BlockStatePredictionManager.mapping [hunk 1]
```diff
@@ -0,0 +1,22 @@
+CLASS net/minecraft/unmapped/C_czisrdmd net/minecraft/client/multiplayer/prediction/BlockStatePredictionManager
+	FIELD f_bmpbngyn predicting Z
+	FIELD f_hcvpxtyt sequence I
```
Comment (supersaiyansubtlety) [inline mappings/net/minecraft/client/multiplayer/prediction/BlockStatePredictionManager.mapping:3]: Is this `int` literally a sequence?
AFAICT it's more like a `sequenceIndex`: it dictates the order of a thing that's part of a sequence.

File: mappings/net/minecraft/client/util/telemetry/event/GameLoadTimesEvent.mapping [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+	METHOD m_szrrcpry (Lnet/minecraft/unmapped/C_tupaamrk$C_ivdfigoc;Lnet/minecraft/unmapped/C_vykgtyww;Lcom/google/common/base/Stopwatch;)V
+		ARG 1 measurement
+		ARG 2 timer
+	METHOD m_wagporif (Lcom/google/common/base/Stopwatch;Lnet/minecraft/unmapped/C_vykgtyww;)Lcom/google/common/base/Stopwatch;
+		ARG 1 property
+	METHOD m_xjunibsk (Lnet/minecraft/unmapped/C_tupaamrk$C_ivdfigoc;J)V
+		ARG 1 time
+	METHOD m_yddbcfgf beginStep (Lnet/minecraft/unmapped/C_vykgtyww;Ljava/util/function/Function;)V
+		ARG 1 measurement
+		ARG 2 timerCreator
```
Comment (supersaiyansubtlety) [inline mappings/net/minecraft/client/util/telemetry/event/GameLoadTimesEvent.mapping]:
```suggestion
		ARG 2 timerFactory
```
?

File: mappings/net/minecraft/util/SegmentedAnglePrecision.mapping [hunk 3]
```diff
@@ -1,3 +1,22 @@
 CLASS net/minecraft/unmapped/C_bhznmlir net/minecraft/util/SegmentedAnglePrecision
+	FIELD f_errgtmjh precision I
+	FIELD f_mqmrqvqi degreeToAngle F
```
Comment (supersaiyansubtlety) [inline mappings/net/minecraft/util/SegmentedAnglePrecision.mapping]:
```suggestion
	FIELD f_mqmrqvqi degreesToAngle F
```
or is there some reason this and the following fields use the singlular `degree`?

File: mappings/net/minecraft/util/SegmentedAnglePrecision.mapping [hunk 4]
```diff
@@ -1,3 +1,22 @@
...
+	FIELD f_ponlgpot angleToDegree F
```
Comment (supersaiyansubtlety) [inline mappings/net/minecraft/util/SegmentedAnglePrecision.mapping]:
```suggestion
	FIELD f_ponlgpot angleToDegrees F
```

**Review by supersaiyansubtlety — COMMENTED** [review]
> discussion question: should we consider breaking up the "every block is in net.minecraft.block" convention for absolutely wild stuff like putting sign blocks in net.minecraft.block.sign?

Sounds good to me.

File: mappings/net/minecraft/client/multiplayer/prediction/BlockStatePredictionManager.mapping (hunk 1 above)
Comment (eristhea) [reply mappings/net/minecraft/client/multiplayer/prediction/BlockStatePredictionManager.mapping:3]: the `sequence` name goes all the way down to `blockChangeSequence` in `ServerPlayNetworkHandler` through a fair few fields and classes, so I'm not going to address it in this pr

I do see your point though, maybe make an issue?

**Comment by eristhea** [comment]
> > discussion question: should we consider breaking up the "every block is in net.minecraft.block" convention for absolutely wild stuff like putting sign blocks in net.minecraft.block.sign?
> 
> Sounds good to me.

this is now implemented!

**Review by EnnuiL — APPROVED** [review]

**Review by supersaiyansubtlety — APPROVED** [review]

**Review by EnnuiL — CHANGES_REQUESTED** [review]
wait oh right; Javadoc's mad at you for missing references; should be an easy fix :p

**Review by supersaiyansubtlety — APPROVED** [review]

**Review by EnnuiL — APPROVED** [review]

**Pull request merged**